pkg/apiserver/manager/hwameistor/storageclass_controller.go

```diff
@@ -0,0 +1,113 @@
+package hwameistor
+
+import (
+	"context"
+	"fmt"
+	hwameistorapi "github.com/hwameistor/hwameistor/pkg/apiserver/api"
+	log "github.com/sirupsen/logrus"
+	v1 "k8s.io/api/core/v1"
+	storagev1 "k8s.io/api/storage/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	v12 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/tools/record"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+const Hwameistor = "lvm.hwameistor.io"
+
+type StorageClassController struct {
+	client.Client
+	record.EventRecorder
+}
+
+func NewStorageClassController(client client.Client, recorder record.EventRecorder) *StorageClassController {
+	return &StorageClassController{
+		Client:        client,
+		EventRecorder: recorder,
+	}
+}
+
+func (scController *StorageClassController) ListHwameistorStroageClass(queryPage hwameistorapi.QueryPage) ([]*hwameistorapi.StorageClass, error) {
+	scList := &storagev1.StorageClassList{}
+	if err := scController.Client.List(context.TODO(), scList); err != nil {
+		log.WithError(err).Error("Failed to list StorageClass")
+		return nil, err
+	}
+	log.Infof("listStorageClass queryPage = %v", queryPage)
+
+	var scs []*hwameistorapi.StorageClass
+	for _, item := range scList.Items {
+
+		if item.Provisioner == Hwameistor {
```

Comment:
we have 2 types of provisioner, `disk.hwameistor.io` may be filtered out here. Is it a expected result or not